### PR TITLE
kk testing comments

### PR DESCRIPTION
**Pull request by kkruel8100**

### CODE REVIEW

**Comment by gkruel** [comment]
No soup for you!

**Comment by kkruel8100** [comment]
I updated code. Please re-review.

**Comment by kkruel8100** [comment]
testing this comment

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 Editing the read me.
 Testing approving KK PR.
+Testing comments.
```
Comment (kkruel8100) [inline README.md:5]: look at this line

**Review by gkruel — APPROVED** [review]

**Pull request merged**